functional.py

```diff
@@ -0,0 +1,83 @@
+# Below you will find several functions I attempted to invent
+
+## 1. sequential_map
+### This functions takes any amount of functions and a container as an input
+### It returns a list of values from initial container modified by consequent use of functions from input
+
+def sequential_map(*args):
+    def apply(func, container):
+        return [func(item) for item in container]
+    data = args[-1]
+    for func in args[:(len(args)-1)]:
+        data = apply(func, data)
+    return data
+
+## 2. consensus_filter
+### This functions takes any amount of functions (which return True or False) and a container as an input
+### It returns a list of values from initial container which are returned with True from functions from input
+
+def consensus_filter(*args):
+    data_changed = dict.fromkeys(args[-1], [])
+    res = []
+    funcs = []
+    for func in args[:len(args)-1]:
+        funcs += [func]
+    def funappl(*args):
+        return [func(args[-1]) for func in funcs]
+    data = args[-1]
+    for el in data:
+        elem = el
+        data_changed[elem] = funappl(funcs, elem)
+    for key in data_changed:
+        if False not in data_changed[key]:
+            res += [key]
+    return res
```

Comment:
Тут что-то мега-переусложнённое, тем не менее, каким-то чудом работает)

```suggestion
    *functions, data = args
    for function in functions:
        data = filter(function, data)
    return list(data)
```